devel/tinyobjloader/Portfile

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           cmake 1.1
+PortGroup           github 1.0
+
+name                tinyobjloader
+categories          devel graphics games
+platforms           darwin
+license             MIT
+maintainers         {outlook.de:judaew @judaew} openmaintainer
+
+description         Tiny but powerful single file wavefront obj loader
+long_description    \
+    Tiny but powerful single file wavefront obj loader written in C++03. No \
+    dependency except for C++ STL. It can parse over 10M polygons with \
+    moderate memory and time. tinyobjloader is good for embedding .obj loader \
+    to your (global illumination) renderer.
+
+if {${subport} eq ${name}} {
+    conflicts           tinyobjloader-devel
+    github.setup        tinyobjloader tinyobjloader 1.0.7 v
+    revision            0
+
+    checksums           rmd160 3cef910b680b65d4812ad3bd7b0f30e6b489527c \
+                        sha256 7a64832ac9b1b3fa98997a0d08b4fc0d07ab8ec54459f411ed080375ef66f91c \
+                        size 993325
+}
```

Comment:
I think this part should be just the the "normal" Portfile and then you just define the "subport" as you do below. There shouldn't be a reason to have this `if` statement as far as I can tell. You can then remove the `name` field as well.

Reply:
Thanks so much